src/languages/en.ts

```diff
@@ -4343,6 +4358,23 @@ const translations = {
                 andEnableWorkflows: 'and enable workflows, then add approvals to unlock this feature.',
             },
         },
+        planTypePage: {
+            planTypes: {
+                team: {
+                    label: 'Collect',
+                    description: 'For teams looking to automate their processes.',
+                },
+                corporate: {
+                    label: 'Control',
+                    description: 'For organizations with advanced requirements.',
+                },
+            },
+            description: "Choose a plan that's right for you. For a detailed list of features and pricing, check out our",
+            subscriptionLink: 'plan types and pricing help page',
+            lockedPlanDescription: ({subscriptionUsersCount, annualSubscriptionEndDate}: WorkspaceLockedPlanTypeParams) =>
+                `You've committed to ${subscriptionUsersCount} active users on the Control plan until your annual subscription ends on ${annualSubscriptionEndDate}. You can switch to pay-per-use subscription and downgrade to the Collect plan starting ${annualSubscriptionEndDate} by disabling auto-renew in`,
+            subscriptions: 'Subscriptions',
+        },
```

Comment:
these translations confirmed here https://expensify.slack.com/archives/C01GTK53T8Q/p1731048615013379?thread_ts=1730105986.836259&cid=C01GTK53T8Q

Reply:
Can we resolve this conversation?